static/js/pages/instances/InstanceList.tsx

```diff
@@ -92,8 +111,19 @@ const InstanceList: FC = () => {
       sortKey: "snapshots",
       className: "u-align--center",
     },
-    { content: "" },
-  ];
+    {
+      content: (
+        <TableColumnsSelect
+          columns={["Status", "Name", "Type", "IPv4", "IPv6", "Snapshots"]}
```

Comment:
I'd extract these to a global, enum-like constant that we can use to reference the headers/aria-labels wherever they are used, so that if by any chance in the future we change them, they get automatically updated everywhere.

Reply:
Good point, this was very unstable before, the constants make the dependency clear now.